src/main/scala/IdentityClientBuilder.scala

```diff
@@ -0,0 +1,21 @@
+package io.blindnet.identityclient
+
+import cats.effect.*
+import org.http4s.*
+import org.http4s.blaze.client.*
+import org.http4s.client.*
+import org.http4s.implicits.*
+
+case class IdentityClientBuilder (
```

Comment:
Remove the class. See comment [here](https://github.com/blindnet-io/privacy-computation-engine/pull/20#discussion_r999143874).

Reply:
Shouldn't we keep both the options of providing the client, and of using the default one? With something like `IdentityClientBuilder.withClient(client).resource`

Reply:
I've reconsidered it and yes, keep them both (`.withClient(client)` and `.withDefaultClient`).